src/averageAges.js

```diff
@@ -14,12 +14,24 @@
  *
  * @return {number}
  */
+
+const calculateAverageAge = (age, people) => {
+  return age.reduce((a, b) => a + b) / people.length;
+};
+
+const getAge = (people) => people.map(person => person.died - person.born);
```

Comment:
1. Name `getAge ` seems like function should return a number, not an numbers array as it is now. 
Use something like `getOnlyAges `, `transformToAges`, `convertToAgesArray`, ...etc
2. You can use destructurization and take only properties you need
`const getAge = (people) => people.map(({ died, born }) => died - born);`